dev/run-server-image.sh

```diff
@@ -37,6 +38,6 @@ docker run "$@" \
   --rm \
   -e SRC_LOG_LEVEL=dbug \
   -e DEBUG=t \
-  --volume $DATA/config:/etc/sourcegraph \
-  --volume $DATA/data:/var/opt/sourcegraph \
+  --volume "$DATA/config:/etc/sourcegraph" \
+  --volume "$DATA/data:/var/opt/sourcegraph" \
```

Comment:
why don't we just use anonymous volumes, then when its done running those volumes are removed? Or do we have data in "$DATA/" that we want to read afterwards/we add data beforehand?
```suggestion
  --volume "/etc/sourcegraph" \
  --volume "/var/opt/sourcegraph" \
```

Reply:
@keegancsmith TIL, thanks! I suspect that's because this script has/is being used locally, hence the prompt earlier in the script asking if it should delete the volumes or not.

I'll dig into it later, that very well may be an outdated use case 🙏